commodore/config.py

```diff
@@ -7,7 +8,13 @@ class Config:
     # pylint: disable=too-many-arguments
     def __init__(self, api_url, api_token, global_git, customer_git, verbose):
         self.api_url = api_url
-        self.api_token = api_token
+        self.api_token = None
+        if api_token is not None:
+            p = P(api_token)
```

Comment:
Won't this raise an exception or anything if it's not a legal path (i.e. illegal characters in the token)?

Reply:
yep it does, when the token would contain a NULL byte :). I'll wrap it in some exception handling

Reply:
on the other hand, `click`'s argument handling breaks when the token contains a null byte,

Reply:
pathlib's Path does not choke on null bytes, I experimented a bit and arrived at the following minimal example: ```from pathlib import Path as P
test="test\u0000blubb"
print(len(test))
print(repr(test))
p = P(test)
print(p)
print(p.is_file())```